es6-shim.js

```diff
@@ -593,53 +685,58 @@
   }
 
   var ArrayShims = {
-    from: function from(iterable) {
+    from: function from(items) {
```

Comment:
Updating the iterator helpers to match the spec requires changing Promise.race, Promise.all, and Array.from -- those are the three places where iterators are used.  It's very awkward to change one without changing the others.